modular/signer/signer_client.go

```diff
@@ -245,34 +275,49 @@ func (client *GreenfieldChainSignClient) RejectUnSealObject(
 		GasLimit: client.gasLimit,
 		Nonce:    nonce,
 	}
-	resp, err := client.greenfieldClients[scope].BroadcastTx(ctx, []sdk.Msg{msgRejectUnSealObject}, txOpt)
 
-	if err != nil {
-		log.CtxErrorw(ctx, "failed to broadcast tx", "err", err, "seal_info", msgRejectUnSealObject.String())
-		if strings.Contains(err.Error(), "account sequence mismatch") {
-			// if nonce mismatch, reset nonce by querying the nonce on chain
-			nonce, err = client.greenfieldClients[scope].GetNonce()
-			if err != nil {
-				log.CtxErrorw(ctx, "failed to get seal account nonce", "err", err, "seal_info", msgRejectUnSealObject.String())
-				return nil, ErrRejectUnSealObjectOnChain
+	var (
+		resp   *tx.BroadcastTxResponse
+		txHash []byte
+	)
+	for i := 0; i < BroadcastTxRetry; i++ {
```

Comment:
This `txOpt` initialization needs to be inside the for loop, because nonce maybe increase.

Reply:
fixed